src/Security/Jwks/Exporter/JwksExporter.php

```diff
@@ -51,7 +51,7 @@ public function export(string $keySetName): array
                 function (KeyChainInterface $keyChain): array {
                     return $this->exporter->export($keyChain);
                 },
-                $this->repository->findByKeySetName($keySetName)
+                array_values($this->repository->findByKeySetName($keySetName))
```

Comment:
just a question, isn't it easier to not return identifiers in KeyChain array keys from the repository

Reply:
we may need them form repo output, to avoir to have to iterate over

in, in the JWKS output, I need NOT to use the keys to comply to JWKS specs, it was a mistake before that I made :/